api/v1alpha1/timeout_types.go

```diff
@@ -0,0 +1,26 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+import gwapiv1 "sigs.k8s.io/gateway-api/apis/v1"
+
+// ConnectionTimeouts defines configuration for timeouts related to connections.
+type ConnectionTimeouts struct {
+	// The timeout for new network connection establishment, including TCP and TLS handshakes.
+	//
+	// +optional
```

Comment:
defaults should be mentioned here

Reply:
Are you referring to Envoy defaults, Envoy Gateway defaults or both? 

`connectTimeout` was customized by Envoy Gateway in #1068 to 10s. 
Do you think that we should:
* Make that more explicit and use a kubebuilder `default` marker for that here? 
* Remove the Envoy Gateway custom value and keep the Envoy default, now that this option is configurable for end users? 
* Just document the Envoy Gateway default and keep it as a translation layer constant? 

`idleTimeout` has an envoy default of 1hr. Should we just document that, or actually reflect that with a kubebuilder marker?

Reply:
comment should be enough here imo, & lets reflect EG defaults